scripts/Start-SasNetworkSurveyDelta.ps1

```
@@ -0,0 +1,352 @@
+    if (-not (Test-Path -LiteralPath $statePath -PathType Leaf)) { return $null }
+    try { return Get-Content -LiteralPath $statePath -Raw -Encoding UTF8 | ConvertFrom-Json }
+    catch { throw "Operator state is unreadable. Use Reset from the launcher menu. State: $statePath" }
+}
+
+function Write-OperatorState {
+    param([Parameter(Mandatory = $true)]$State)
+    $State.updated_at = [datetimeoffset]::Now.ToString('o')
+    $State | ConvertTo-Json -Depth 6 | Set-Content -LiteralPath $statePath -Encoding UTF8
+}
+
+function Get-InputCandidates {
```

PSScriptAnalyzer (1.25.0)

[warning] The cmdlet 'Get-InputCandidates' uses a plural noun. A singular noun should be used instead.

Suggested fix: Singularized correction of 'Get-InputCandidates'

(PSUseSingularNouns)